commit 57a68dc342dd
Author: Henrik Skupin <mail@hskupin.info>
Date:   Thu Jun 7 15:48:48 2018 +0200

    Bug 1334981 - Backed out changeset d6ca24ba3673 for regression caused by bug 887718. r=enndeakin, a=RyanVM
    
    MozReview-Commit-ID: 9hIcRVRseO1
    
    --HG--
    extra : source : 1f16a2ff45ab492025e42afb8122f4d52cb24c0b
---
 dom/base/nsFocusManager.cpp | 4 +---
 1 file changed, 1 insertion(+), 3 deletions(-)

diff --git dom/base/nsFocusManager.cpp dom/base/nsFocusManager.cpp
index 864c76402cb12..c5758cfabd523 100644
--- dom/base/nsFocusManager.cpp
+++ dom/base/nsFocusManager.cpp
@@ -742,9 +742,7 @@ nsFocusManager::WindowRaised(mozIDOMWindowProxy* aWindow)
       return NS_ERROR_FAILURE;
     }
 
-    if (!sTestMode) {
-      baseWindow->SetVisibility(true);
-    }
+    baseWindow->SetVisibility(true);
   }
 
   // If this is a parent or single process window, send the activate event.
